applications/crossbar/test/cb_alerts_tests.erl

```diff
@@ -28,45 +30,33 @@ cb_alerts_test_() ->
 %% @doc
 %% @end
 %%--------------------------------------------------------------------
--spec check_port_requests() -> [{string(), boolean()}].
-check_port_requests() ->
-    Mod = 'knm_port_request',
-    ToMeck = ['kz_services_reseller', Mod],
-    _ = lists:foreach(fun(M) -> meck:new(M, [passthrough]) end, ToMeck),
-    meck:expect('kz_services_reseller', 'is_reseller', fun(_) -> 'false' end),
-
+-spec do_check_port_requests() -> [{string(), boolean()}].
+do_check_port_requests() ->
     Context = cb_context:set_resp_data(cb_context:new(), []),
-    {'ok', ActivePorts} = AccountActivePorts = account_active_ports(),
+    {'ok', ActivePorts} = account_active_and_unconfirmed_ports(),
 
     %% 1 submitted, 1 unconfirmed, and 1 rejected.
-    meck:expect(Mod, 'account_active_ports', fun(_) -> AccountActivePorts end),
-    Context1 = cb_alerts:check_port_requests(Context),
-    [Alert1, Alert2] = cb_context:resp_data(Context1),
+    Context1 = cb_alerts:do_check_port_requests(ActivePorts, Context),
+    Resp = cb_context:resp_data(Context1),
```

Comment:
Same here, pattern-match: `[Alert1, Alert2] = Resp = cb_context:resp_data(Context1),` or get rid of Resp if unused.